code/modules/mining/machine_redemption.dm

```diff
@@ -156,6 +156,8 @@
 	else
 		return
 
+	SEND_SIGNAL(src, COMSIG_ORM_COLLECTED_ORE)
```

Comment:
for this signal, it would make sense to pass in the ore as one of the arguments, even if you don't end up using it.